src/io/json.c

```diff
@@ -102,46 +102,35 @@
 #define JSON_BASE_TEMPLATE_END \
     "}\n"
 
-static INLINE bool is_disabled(struct criterion_test *t, struct criterion_suite *s) {
-    return t->data->disabled || (s->data && s->data->disabled);
-}
 
 static CR_INLINE
-const char *get_status_string(struct criterion_test_stats *ts,
-                              struct criterion_suite_stats *ss) {
-
-    const char *status = "PASSED";
-    if (ts->crashed || ts->timed_out)
-        status = "ERRORED";
-    else if (ts->failed)
-        status = "FAILED";
-    else if (is_disabled(ts->test, ss->suite))
-        status = "SKIPPED";
-    return status;
+const char *get_status_string(struct criterion_test_stats *ts){
```

Comment:
I stand corrected. I think the project really needs something to enforce a coding style.